ocdskingfisherprocess/database.py

```diff
@@ -41,14 +41,18 @@ def __init__(self, config):
                                                    nullable=False, default=False),
                                          sa.Column('transform_from_collection_id', sa.Integer,
                                                    sa.ForeignKey("collection.id"), nullable=True),
-                                         sa.Column('transform_type', sa.Text, nullable=True),
+                                         sa.Column('transform_type', sa.Text, nullable=False),
                                          sa.Column('deleted_at', sa.DateTime(timezone=False), nullable=True),
                                          sa.Column('cached_releases_count', sa.Integer, nullable=True),
                                          sa.Column('cached_records_count', sa.Integer, nullable=True),
                                          sa.Column('cached_compiled_releases_count', sa.Integer, nullable=True),
                                          sa.UniqueConstraint('source_id', 'data_version', 'sample',
```

Comment:
This should be removed?

Reply:
I think we agreed to have a unique constraint on those three fields, as otherwise we can't determine which collection to use for storing data from Scrapy.

Reply:
> Scrape sends data to Process with 3 flags ("collection_source", "collection_data_version", "collection_sample") and Process must identify the correct ID of collection to put the data into. So we somehow need a unique index on those 3 flags for collections that are not transforms.

Reply:
Ahh, nevermind, I see the unwanted lines now.